contracts/hyphen/LiquidityProviders.sol

```diff
@@ -118,6 +123,22 @@ contract LiquidityProviders is
         lpToken = ILPToken(_lpToken);
     }
 
+    function increaseCurrentLiquidity(address tokenAddress, uint256 amount) public onlyLiquidityPool {
+        _increaseCurrentLiquidity(tokenAddress, amount);
+    }
+
+    function decreaseCurrentLiquidity(address tokenAddress, uint256 amount) public onlyLiquidityPool {
+        _decreaseCurrentLiquidity(tokenAddress, amount);
+    }
+
+    function _increaseCurrentLiquidity(address tokenAddress, uint256 amount) private {
```

Comment:
We should emit an event whenever the current liquidity is being changed. This will be required if we want to create a graph to track liquidity over time

Reply:
will do